earthpy/spatial.py

```diff
@@ -627,6 +627,34 @@ def hillshade(arr, azimuth=30, altitude=30):
     return 255 * (shaded + 1) / 2
 
 
+def crs_check(path):
+    """Get the CRS of a raster file from a file path.
+
+    Parameters
+    ----------
+    path : string
+        Path to the raster file.
```

Comment:
Probably want to say `Path to a raster file in a format that rasterio can read.`